nuxt.config.js

```diff
@@ -166,6 +166,11 @@ const config = {
         path: '/*',
         component: 'pages/index.vue'
       });
+      routes.push({
```

Comment:
Get this working as intended with:
```diff
diff --git a/nuxt.config.js b/nuxt.config.js
index 55a5e9a5..8638b281 100644
--- a/nuxt.config.js
+++ b/nuxt.config.js
@@ -168,8 +168,8 @@ const config = {
       });
       routes.push({
         name: 'collections',
-        path: '/collections',
-        component: 'pages/collections/_type/_.vue'
+        path: '/(collections)',
+        component: 'pages/index.vue'
       });
     },
     linkExactActiveClass: 'exact-active-link'
```

Reply:
@rwd thanks, the collections foyer page now works as expected, however, visiting https://localhost:3001/en/collections/topics

I see the error:
Error: Unsupported Entity type: undefined, should be one of: agent, concept, place, organization, timespan!

Reply:
Can you double-check this? I see no such error.

Reply:
I cannot replicate the error today either